.travis.yml

```
@@ -4,7 +4,8 @@ os:
   - linux
   - osx
 julia:
-  - 1.0
+  - 1.3
+  - 1.4
```

Maybe you want to use
```suggestion
  - 1
```
to ensure that the latest stable release of Julia is tested, without having to update it manually.